### DISCUSSION TITLE
Remove default page background gradient from HASP theme?

### DISCUSSION THREAD

**Discussion by fvanroie** [HASwitchPlate/openHASP#647] [Polls]
There has been a [suggestion to remove the default gradient](https://github.com/HASwitchPlate/openHASP/pull/581) from the pages background, to make it look more modern.

Because this change could *"break layouts"* I consider this a *"breaking change"*.
Therefor I would like to poll if this change is worth it to go from this:

![287360332-8fdf748e-8840-42d2-91a3-32bfeda9e6db](https://github.com/HASwitchPlate/openHASP/assets/15969459/b11dd259-61f1-4422-a121-add0c640d2ea)

to this:
![286657149-25435bba-54ba-4a95-a099-86d6088a0d54](https://github.com/HASwitchPlate/openHASP/assets/15969459/d5837e78-d5b1-45a8-bded-82739a1b9902)

**Comment by kuba2k2** [comment]
It could be less of a breaking change, if the new background would look more like this:
![obraz](https://github.com/HASwitchPlate/openHASP/assets/30433568/de361289-ab03-4910-8be6-5fedb4dac76e)
However, I don't know if this is what the OP desired.

**Comment by zalves** [comment]
I know that there has to be progress. But that gradient will always be missed as in the old days.

**Comment by vogt31337** [comment]
would it be possible to make it optional aka configurable? So that everyone could just use their desired background color /gradient?

**Reply by fvanroie** [reply]
Customizable themes are not on my to-do list at this time.
